terraform/modules/self-service/modules/cognito/cognito.tf

```
@@ -2,9 +2,94 @@ locals {
   service = "self-service"
 }
 
+resource "aws_sns_topic" "cognito-sns-topic" {
```

```suggestion
resource "aws_sns_topic" "cognito_sns_topic" {
```
we should be consistent with our naming convention, i.e. resource names use underscores...